src/saving/SaveManagerGUI.cs

```diff
@@ -103,8 +107,19 @@ public override void _Ready()
         deleteOldConfirmDialog = GetNode<CustomConfirmationDialog>(DeleteOldConfirmDialogPath);
         saveDirectoryWarningDialog = GetNode<CustomConfirmationDialog>(SaveDirectoryWarningDialogPath);
         errorSaveDeletionFailed = GetNode<CustomConfirmationDialog>(SaveDeletionFailedErrorPath);
+        filterOptions = GetNode<OptionButton>(FilterOptionsPath);
 
         saveList.Connect(nameof(SaveList.OnItemsChanged), this, nameof(RefreshSaveCounts));
+
+        filterOptions.AddItem(TranslationServer.Translate("SAVE_MANUAL"), (int)SaveInformation.SaveType.Manual);
```

Comment:
Using enum values and getting the thing to translate like (example from different context):
```c#
        foreach (var mode in new[] { FogOfWarMode.Ignored, FogOfWarMode.Regular, FogOfWarMode.Intense })
        {
            fogOfWarModeDropdown.AddItem(
                TranslationServer.Translate(mode.GetAttribute<DescriptionAttribute>().Description), (int)mode);
        }
```

this also means that the `3` value below may not be hardcoded as adding new save types would conflict.